data/scripting/infrastructure.lua

```diff
@@ -218,13 +245,19 @@ end
 -- RST
 -- .. function:: check_trees_rocks_poor_hamlet(player, sf, warehouse, waretable_rocks, waretable_trees)
 --
---    Checks for rocks or trees in the region of the player starting field and adds wares if no immovables found
+--    Checks for rocks or trees in the region of the player starting field
+--    and adds wares to the warehouse if no immovables were found
 --
 --    :arg player: the player to check
+--    :type player: :class:`wl.game.Player`
 --    :arg sf: starting field of the player
---    :arg warehouse: The type of warehouse (string) to add wares to
---    :arg waretable_rocks: a table of pairs {ware = value} to add if no rocks
---    :arg waretable_trees: a table of pairs {ware = value} to add if no trees
+--    :type sf: :attr:`starting_field`
+--    :arg warehouse: The type of warehouse to add wares to
+--    :type warehouse: :const:`string`
+--    :arg waretable_rocks: a table of pairs {:const:`"name"` = value} to add if no rocks
```

Comment:
```suggestion
--    :arg waretable_rocks: a table of pairs {:const:`"name"` = value} to add if no rocks were found nearby
```
same for `waretable_trees`

Reply:
uups, those where stolen some how

Reply:
done